docs/root/version_history/current.rst

```diff
@@ -29,6 +29,7 @@ Removed Config or Runtime
 
 New Features
 ------------
+* http: introduced new HTTP/1 and HTTP/2 codec implementations that will remove the use of exceptions for control flow due to high risk factors and instead use error statuses. The old behavior is deprecated, but can be used during the removal period by disabling the runtime feature `envoy.reloadable_features.new_codec_behavior`. The removal period will be one month.
```

Comment:
generally go for "setting the runtime feature X to false" as it's a bit more clear how one disables.

Reply:
done!